tests/py/test_api.py

```diff
@@ -79,7 +79,6 @@ def test_is_colored_output_on(self):
 def test_suite():
     suite = unittest.TestSuite()
     suite.addTest(TestCompile.parametrize(TestCompile, model=models.resnet18(pretrained=True)))
-    suite.addTest(TestCompile.parametrize(TestCompile, model=models.resnet50(pretrained=True)))
```

Comment:
removing this since one variant of resnet (resnet 18) already exists ?

Reply:
yeah